sql/flow_config.go

```diff
@@ -119,5 +119,8 @@ func GetPythonSDKComptability(configURL, sqlCliVersion string) (astroRuntimeVers
 	}
 
 	SQLCLICompatibilityVersions := resp.Compatibility[sqlCliVersion]
+	if SQLCLICompatibilityVersions.AstroRuntime == "" {
+		return "", "", fmt.Errorf("could not find a matching SQL CLI compatibility version: current version %s version map: %v  %w", sqlCliVersion, resp.Compatibility, err)
+	}
 	return SQLCLICompatibilityVersions.AstroRuntime, SQLCLICompatibilityVersions.AstroSdkPython, nil
```

Comment:
What do you think if we followed the same strategy we have at the moment for the `astroCliCompatiblity` section?

It feels like both are very similar, but we are dealing with them in different ways. Do we have exact matches for  `compatibility`? It would be great if we could normalise the behaviour of both of them, probably having a `default` section in `compatibility` as well

Reply:
I am happy with both approaches. Since we control the releases for SQL CLI, we could make sure that we add an entry for the release in the compatibility matrix. If not, do we know what default value to specify for the runtime image and python-sdk version irrespective of the version for SQL-CLI?